### PR TITLE
Assignment 3 ( Arithmetic Operators , Patterns , even and odd numbers , prime numbers , Fibonaaci Series )

### PR DESCRIPTION
**Pull request by rishabh1164**
Solved Problems based on Arithmetic operators ,  even and odd numbers  and wrote programs to find prime numbers upto n.
Wrote Program for printing FIbonaaci Series .

### CODE REVIEW

File: Assignment2.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
+//Question 1
```
Comment (ajay-nx) [inline Assignment2.js:2]: This pull request should have only Assignment 3 changes. Please update the PR accordingly.

File: Assignment3.js [hunk 2]
```diff
@@ -0,0 +1,349 @@
+// Question 1 - Write a program to perform functionality of a calculator (add,sub,multiply,divide)
+// created calculator arrow function which takes two numerical values and one character value ( that tells us which arithmetic operation needs to perform)
```
Comment (ajay-nx) [inline Assignment3.js:2]: In this problem - It's better to use switch statement for more readability & maintainability.

Comment (ajay-nx) [inline Assignment3.js:2]: **Indentation** of this file isn't proper & you can use **multiline comments** for better readability.

**Review by ajay-nx — CHANGES_REQUESTED** [review]

File: Assignment2.js (hunk 1 above)
Comment (rishabh1164) [reply Assignment2.js]: updated PR ( only have assignment3 )

File: Assignment3.js (hunk 2 above)
Comment (rishabh1164) [reply Assignment3.js]: used eslint and prettier for better indentation.
used multiline comments wherever needed .

**Review by ajay-nx — APPROVED** [review]

**Pull request merged**